src/app/component/key/key-interface.js

```diff
@@ -0,0 +1,294 @@
+import './key-interface.css';
+import './key-interface.media.css';
+import createElement from '../../utils/create-element';
+import TyperInterface from './typer-interface';
+
+const CssClasses = {
```

Comment:
I would not recommend you to use such pattern. You can create constants, if class names are reusable, but putting all of them in such object, even is they are used only one time, or not used at all is an overhead IMHO.